.gitignore

```diff
@@ -1,3 +1,5 @@
+/.idea
```

Comment:
I'm personally against those kind of entries in .gitignore, it's developer specific and should go to [global gitignore](https://docs.github.com/en/get-started/getting-started-with-git/ignoring-files#configuring-ignored-files-for-all-repositories-on-your-computer)

Reply:
sure, will remove them :+1:

Reply:
fixed